app/views/docs/functions.phtml

```diff
@@ -279,6 +279,30 @@ void main() { // Init SDK
             </td>
             <td>Your function event payload. This value is available only when your function trigger is 'event'. This variable value contains a string in JSON format with your specific event data.</td>
         </tr>
+        <tr>
+            <td>
```

Comment:
We should add a manual note that this vars have been introduced in v0.8.

Here is an example:
https://github.com/appwrite/docs/blob/main/app/views/docs/environment-variables.phtml#L57-L59

Reply:
![functionenvdocs](https://user-images.githubusercontent.com/9708641/112201911-26d2fd80-8be7-11eb-9203-1f84d21a0656.png)